docs/tutorials/web/index.md

```diff
@@ -64,9 +66,11 @@ For more detail on the steps below, see the [build a web application](../../refe
 
    Bootstrap your web application according in your web framework of choice e.g. vuejs, reactjs, angularjs.
 
-   a. Add the ONNX Runtime dependency
-   b. Consume the onnxruntime-web API in your application
-   c. Add pre and post processing appropriate to your application and model
+   1. [Add the ONNX Runtime dependency](../../reference/build-web-app.md#add-onnx-runtime-web-as-dependency)
+
+   1. Consume the onnxruntime-web API in your application
+
+   1. Add pre and post processing appropriate to your application and model
```

Comment:
Link to '../../reference/build-web-app.html#pre-and-post-processing'

Reply:
Fixed